content/docs/configuration-services.md

```diff
@@ -119,12 +120,12 @@ export const ArithmeticsModule: Module<ArithmeticsServices, PartialLangiumServic
 ```
 The services which `ServiceClass` depends on need to be registered in the constructor:
 ```Typescript
-export class ServiceClass{
-    readonly serviceOne: ServiceOne;
-    readonly serviceTwo: ServiceTwo;
-    readonly serviceN: ServiceN;
+export class ServiceClass {
+    private readonly serviceOne: ServiceOne;
+    private readonly serviceTwo: ServiceTwo;
+    private readonly serviceN: ServiceN;
```

Comment:
Having `private` on service fields makes it difficult to extend the service class. Even we don't expect end users to create a service class inheritance structure, I believe it to be good practice to have them in the doc as `protected`.

Reply:
I agree from the perspective of the library / framework code. In user code, I tend to make things private by default because I can simply change that in case I need another visibility. And I would like to not encourage using inheritance in user code. The only reason we use it in Langium is to override default functionality.